app/models/cart.rb

```diff
@@ -31,11 +36,21 @@ def subtotal(item_id, price)
   end
 
   def number_of_days
-    @items['days'] ||= 0
+    @items['days'] ||= 1
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.